features/step_definitions/csv_steps.rb

```diff
@@ -0,0 +1,5 @@
+Then /^JavaScript should run without any errors$/ do
+  # If the `js-enabled` class has been set on the <body> element then
+  # JavaScript is running on the page and there are no SRI or similar errors
+  expect(page.find('body')[:class].split(" ")).to include("js-enabled")
```

Comment:
I'm not super knowledgeable about the issues behind this - so this comment might be hopelessly wrong - but I'm not sure how this test helps us with errors.

as I understand (from looking at source) that this line after the opening `<body>` element: `<script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>` modifies the className and that will run regardless (and before) any external javascript files which have an integrity hash.

Reply:
My mistake. I've changed it to look for something that we add via an external JS file.